src/api.js

```diff
@@ -34,266 +34,13 @@ const getLastUpdated = async (siteName) => {
-) => {
-  const apiEndpoint = getCreatePageApiEndpoint({
-    folderName,
-    subfolderName,
-    newFileName,
-    siteName,
-    resourceName,
-  })
-  const params = { content }
-  await axios.post(apiEndpoint, params)
-
-  // redirect to new page upon successful creation
-  if (folderName) {
-    return `/sites/${siteName}/folders/${folderName}/${
-      subfolderName ? `subfolders/${subfolderName}/` : ""
-    }editPage/${newFileName}` // V2
-  }
-  if (resourceName) {
-    return `/sites/${siteName}/resourceRoom/${resourceRoomName}/resourceCategory/${resourceName}/editPage/${newFileName}`
-  }
-  return `/sites/${siteName}/pages/${newFileName}`
-}
-
-const renamePageData = async (
-  { folderName, subfolderName, fileName, siteName, resourceName, newFileName },
-  content,
-  sha
-) => {
-  const apiEndpoint = getRenamePageApiEndpoint({
-    folderName,
-    subfolderName,
-    fileName,
-    siteName,
-    resourceName,
-    newFileName,
-  })
-  const params = {
-    content,
-    sha,
-  }
-  await axios.post(apiEndpoint, params)
-}
-
-const updatePageData = async (
-  { folderName, subfolderName, fileName, siteName, resourceName },
-  content,
-  sha
-) => {
-  const apiEndpoint = getPageApiEndpoint({
-    folderName,
-    subfolderName,
-    fileName,
-    siteName,
-    resourceName,
-  })
-  const params = {
-    content,
-    sha,
-  }
-  await axios.post(apiEndpoint, params)
-}
-
-const deletePageData = async (
-  { folderName, subfolderName, fileName, siteName, resourceName },
-  sha
-) => {
-  const apiEndpoint = getPageApiEndpoint({
-    folderName,
-    subfolderName,
-    fileName,
-    siteName,
-    resourceName,
-  })
-  const params = { sha }
-  return axios.delete(apiEndpoint, {
-    data: params,
-  })
-}
-
-// Folder and subfolders
-const renameFolder = async ({ siteName, folderName, newFolderName }) => {
-  const apiUrl = `${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/collections/${folderName}/rename/${newFolderName}`
-  return axios.post(apiUrl)
-}
-
-const deleteFolder = async ({ siteName, folderName }) => {
-  const apiUrl = `${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/collections/${folderName}`
-  return axios.delete(apiUrl)
-}
-
-const deleteSubfolder = async ({ siteName, folderName, subfolderName }) => {
-  const apiUrl = `${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/folders/${folderName}/subfolder/${subfolderName}`
-  return axios.delete(apiUrl)
-}
-
-const renameSubfolder = async ({
-  siteName,
-  folderName,
-  subfolderName,
-  newSubfolderName,
-}) => {
-  const apiUrl = `${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/folders/${folderName}/subfolder/${subfolderName}/rename/${newSubfolderName}`
-  return axios.post(apiUrl)
-}
-
 const getResourceRoomName = async (siteName) => {
   const apiUrl = `${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/resource-room`
   const resp = await axios.get(apiUrl)
   const { resourceRoom } = resp.data
   return resourceRoom
 }
```

Comment:
Is this method still used? I think we might be able to remove it as well

Reply:
Oops yup, removed in c142579e!